packages/controller-utils/src/util.ts

```diff
@@ -68,7 +68,7 @@ export function isSafeChainId(chainId: Hex): boolean {
  * @returns A '0x'-prefixed hex string.
  */
 // TODO: Either fix this lint violation or explain why it's necessary to ignore.
-// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Why are we removing the comment to disable the naming-convetion linting rule everywhere?
If we want to disable it, we would need to:
1. fix the offender (in this case the BNToHex not being camelCase.. although it makes sense that it isn't, and in that case keep the comment to disable)
2. remove or update the TODO comment on top.

None of that should be done in this PR. Can you please revert the change that removes the comment that disables the naming-convention linting rule?